web/src/app/projects/repository/repository.component.html

```diff
@@ -1,222 +1,275 @@
+                <mat-icon svgIcon="request_white"></mat-icon>
+
+                <div class="repository-detail-card__header__text" fxFlexOffset="15px" fxFlexAlign="end">Pull Requests</div>
+
+              </h3>
+            </mat-card-title>
+          </mat-card-header>
+          <mat-card-content class="repository-detail-card__content">
+            <mat-list role="list" *ngIf="repository.pullRequests && repository.pullRequests.length > 0; else pullRequestsOther">
+              <mat-list-item role="listitem" *ngFor="let pullRequest of repository.pullRequests | slice:0:5">
+                <mat-divider class="repository-detail-card__content__divider"></mat-divider>
+                <img matListAvatar [src]="pullRequest.owner.avatarUrl" [alt]="pullRequest.owner.username" />
+                <h4 mat-line class="repository-detail-card__content__title">{{ pullRequest.title }}</h4>
+                <p mat-line class="repository-detail-card__content__request-user">{{ pullRequest.owner.username }}</p>
+                <span class="repository-detail-card__content__time-container-bottom">
+                  <mat-icon svgIcon="time_icon"></mat-icon>
+                  <p mat-line class="repository-detail-card__content__time">{{ pullRequest.createdOn | timeAgo }}</p>
+                </span>
+
               </mat-list-item>
             </mat-list>
-
           </mat-card-content>
         </mat-card>
+        <ng-template #pullRequestsOther>
+          <div class="empty"> No pull requests are available</div>
+        </ng-template>
       </div>
-      <div fxFlex="24%">
-        <mat-card>
-          <mat-card-content class="dashboard-card">
-            <h3 *ngIf="repository.pullRequests && repository.pullRequests.length > 0; else pullRequestsOther" class="card-icon heading info-outline">Pull Requests</h3>
-            <ng-template #pullRequestsOther>
-              <p> No pull requests are available</p>
-            </ng-template>
-            <mat-list>
-              <mat-list-item *ngFor="let pullRequest of repository.pullRequests | slice:0:5">
-                <img matListAvatar [src]="pullRequest.owner.avatarUrl" [alt]="pullRequest.owner.username" />
-                <h4 mat-line>{{ pullRequest.title }}</h4>
-                <p mat-line>{{ pullRequest.owner.username }}</p>
-                <p mat-line>{{ pullRequest.createdOn | timeAgo }}</p>
+
+      <div fxFlex="33%" fxFlex.sm="100%" fxFlex.xs="100%">
+        <mat-card class="repository-detail-card">
+          <mat-card-header class="blue repository-detail-card__header">
+            <mat-card-subtitle>
+              <h3 fxLayout fxLayoutAlign="start center">
+                <mat-icon svgIcon="event"></mat-icon>
+                <div class="repository-detail-card__header__text" fxFlexOffset="15px" fxFlexAlign="end">Events</div>
+              </h3>
+              <ng-template #eventsOther>
+                <div class="empty"> No events are available</div>
```

Comment:
Some places have a space before the text and some don't, can we be consistent and have no space

```html
<div class="empty">No events are available</div>
```